src/test/java/uk/gov/ons/census/fwmtadapter/messaging/RefusalReceiverIT.java

```diff
@@ -49,6 +61,10 @@
   @Autowired private RabbitQueueHelper rabbitQueueHelper;
 
   private EasyRandom easyRandom = new EasyRandom();
+  private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
make final?

Reply:
not sure this is needed in test code, it's not acting as a constant, and state isn't important.  I'll do it anyway though